### PR TITLE
Ct fixes

### PR DESCRIPTION
**Pull request by tgraf**

### CODE REVIEW

File: bpf/lib/conntrack.h [hunk 1]
```diff
@@ -387,8 +387,7 @@ static inline int __inline__ ct_create6(void *map, struct ipv6_ct_tuple *tuple,
 	tuple->dport = 0;
 	tuple->flags |= TUPLE_F_RELATED;
 
-	cilium_trace(skb, DBG_CT_CREATED, (tuple->sport << 16) | tuple->dport,
-		     (tuple->nexthdr << 8) | tuple->flags);
+	cilium_trace(skb, DBG_CT_CREATED, 0, (tuple->nexthdr << 8) | tuple->flags);
```
Comment (borkmann) [inline bpf/lib/conntrack.h:390]: Should be: ntohs(tuple->sport) << 16) | ntohs(tuple->dport) instead of 0?

File: bpf/lib/conntrack.h [hunk 2]
```diff
@@ -430,8 +429,7 @@ static inline int __inline__ ct_create4(void *map, struct ipv4_ct_tuple *tuple,
...
```
Comment (borkmann) [inline bpf/lib/conntrack.h:432]: Same here?

File: bpf/lib/conntrack.h (hunk 1 above)
Comment (tgraf) [inline bpf/lib/conntrack.h:390]: We're creating two CT entries. The real one, above this piece of code plus an ICMPv6 entry with `TUPLE_F_RELATED` flag. The latter has the sport/dport bits set to 0.

Comment (borkmann) [inline bpf/lib/conntrack.h:390]: Okay, makes sense.

**Pull request merged**